### ISSUE TITLE
Error: Could not resolve '../util/PropMap' from src/domain/location.js

### ISSUE THREAD

**Issue by brettanomyces** [SolarNetwork/sn-api-core-js#1]
`npm run build` fails with the following message:

> Error: Could not resolve '../util/PropMap' from src/domain/location.js

Cause is incorrect capitalization of `propMap`.

Fix:

```

diff --git a/src/domain/location.js b/src/domain/location.js
index 8e084e1..a1b97f2 100644
--- a/src/domain/location.js
+++ b/src/domain/location.js
@@ -1,4 +1,4 @@
-import PropMap from '../util/PropMap';
+import PropMap from '../util/propMap';
 
 const CountryKey = 'country';
 const ElevationKey = 'elevation';
```

**Comment by msqr** [comment]
Thanks for the fix, it must have come from my workstation's [case-insensitive filesystem](https://en.wikipedia.org/wiki/HFS_Plus)! Would you like to open a PR with this change? Then you'll get credit for the fix in git.

**Comment by msqr** [comment]
Fixed by #2, thanks!
